### ISSUE TITLE
`ssl_verify` `.condarc` option doesn't work if pointing to a custom crt

### ISSUE THREAD

**Issue by dhirschfeld** [mamba-org/mamba#628]
In our linux containers we point the `.condarc` [`ssl_verify`](https://docs.conda.io/projects/conda/en/latest/user-guide/configuration/use-condarc.html#ssl-verification-ssl-verify) option to our custom crt file:
```
ssl_verify: /etc/ssl/certs/ca-certificates.crt
```

`conda` picks this up and we can connect to our internal repo server but it appears `mamba` ignores this setting and we get SSL errors when trying to connect.

```
RuntimeError: Download error (60) SSL peer certificate or SSH remote key was not OK [https://conda.smc.stanwell.com/stanwell/uat/noarch/repodata.json]
```
<br>

If however, we set the `REQUESTS_CA_BUNDLE` environment variable, then `mamba` works fine:
```
export REQUESTS_CA_BUNDLE=/etc/ssl/certs/ca-certificates.crt
```

It would be good if `mamba` would also use the `ssl_verify` `.condarc` option when pointing to a custom crt.

**Comment by dhirschfeld** [comment]
Looks related to https://github.com/mamba-org/mamba/issues/151#issuecomment-617535208

**Comment by wolfv** [comment]
thanks for opening this issue. I will have to check why we're not using the option from condarc.

**Comment by tl-hbk** [comment]
This change here seems to have been what caused the `.condarc` option to stop working

https://github.com/mamba-org/mamba/commit/5c2951cc181d383e131fc345d439fc2878fe94db#diff-83165d5f8a167348297e4c513a32c2e9b78cfcd82f7a134a770683d710073989L95-L98

**Comment by dhirschfeld** [comment]
It looks like the code assumes that `ssl_verify` is either a boolean or an empty string and doesn't admit the possibility to specify a path to `ca-certificates.crt`?

![image](https://user-images.githubusercontent.com/881019/109069600-2a38ad80-773d-11eb-8673-0a0dae31c151.png)

https://github.com/mamba-org/mamba/blob/9997ff826585b7f7ee613714d98e0941986c42f2/mamba/utils.py#L158-L166

**Comment by wolfv** [comment]
this should be fixed by #712. I'll release today or on Monday.
